assets/js/wpr-admin-common.js

```diff
@@ -4,4 +4,62 @@ jQuery( document ).ready( function( $ ){
 		var url = $( this ).attr( 'href' ).replace( 'admin-post', 'admin-ajax' );
 		$.get( url ).done( $( this ).closest( '.notice' ).hide( 'slow' ) );
 	});
+
+	$( '#deactivate' ).click( function() {
+		$( '#export_settings' ).prop( 'checked', false );
+		$( '#export_settings' ).hide();
+		$( 'label[for=export_settings]' ).hide();
+	});
+
+	$( '#safe_mode' ).click( function() {
+		$( '#export_settings' ).show();
+		$( 'label[for=export_settings]' ).show();
+		$( '#export_settings' ).prop( 'checked', true );
+	});
+
+	$( '#wpr-deactivation-intent-form' ).submit(function (e) {
+		const checked = $( '#export_settings' ).prop('checked');
+		if(! checked) {
+			return true;
+		}
+
+		e.preventDefault();
+
+		$.ajax( {
+			url: rocket_option_export.rest_url_option_export,
+			method: 'GET',
+			beforeSend: function ( xhr ) {
+				xhr.setRequestHeader( 'X-WP-Nonce', jQuery('#wpr-deactivation-intent-form input[name="rest_auth_nonce"]').val() );
+			},
+			success: function( data, textStatus, xhr ) {
+				const disposition = xhr.getResponseHeader('content-disposition');
+
+				const filenames = disposition.match('filename="([^"]+)"');
+
+				if(! filenames.length) {
+					return;
+				}
+
+				const filename = filenames.pop();
+
+				const url = URL.createObjectURL( new Blob( [ JSON.stringify(data, null, 2) ], {
```

Comment:
@vmanthos can you check if this line is on the website when you test?

Reply:
Indeed, it was an issue with caching.